From 78a5fc0b82b8cebbec953af0c30a342ef38f2ee3 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Thu, 9 Aug 2018 21:38:40 +0000
Subject: [PATCH] OS-7115 Need Intel ucode update to 20180807 Reviewed by: John
 Levon <john.levon@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 manifest                             |   7 +++++++
 usr/src/data/ucode/README.ucode      |   2 +-
 usr/src/data/ucode/intel/000106A5-01 | Bin 10240 -> 12288 bytes
 usr/src/data/ucode/intel/000106A5-02 | Bin 10240 -> 12288 bytes
 usr/src/data/ucode/intel/000106E5-01 | Bin 7168 -> 9216 bytes
 usr/src/data/ucode/intel/000106E5-02 | Bin 7168 -> 9216 bytes
 usr/src/data/ucode/intel/000106E5-10 | Bin 7168 -> 9216 bytes
 usr/src/data/ucode/intel/00020652-02 | Bin 8192 -> 9216 bytes
 usr/src/data/ucode/intel/00020652-10 | Bin 8192 -> 9216 bytes
 usr/src/data/ucode/intel/00020655-02 | Bin 3072 -> 4096 bytes
 usr/src/data/ucode/intel/00020655-10 | Bin 3072 -> 4096 bytes
 usr/src/data/ucode/intel/00020655-80 | Bin 3072 -> 4096 bytes
 usr/src/data/ucode/intel/000206A7-02 | Bin 12288 -> 12288 bytes
 usr/src/data/ucode/intel/000206A7-10 | Bin 12288 -> 12288 bytes
 usr/src/data/ucode/intel/000206C2-01 | Bin 0 -> 11264 bytes
 usr/src/data/ucode/intel/000206C2-02 | Bin 0 -> 11264 bytes
 usr/src/data/ucode/intel/000206D6-01 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000206D6-04 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000206D6-08 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000206D6-20 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000206D6-40 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000206D7-01 | Bin 19456 -> 19456 bytes
 usr/src/data/ucode/intel/000206D7-04 | Bin 19456 -> 19456 bytes
 usr/src/data/ucode/intel/000206D7-08 | Bin 19456 -> 19456 bytes
 usr/src/data/ucode/intel/000206D7-20 | Bin 19456 -> 19456 bytes
 usr/src/data/ucode/intel/000206D7-40 | Bin 19456 -> 19456 bytes
 usr/src/data/ucode/intel/000206E6-04 | Bin 0 -> 9216 bytes
 usr/src/data/ucode/intel/000206F2-01 | Bin 13312 -> 14336 bytes
 usr/src/data/ucode/intel/000206F2-04 | Bin 13312 -> 14336 bytes
 usr/src/data/ucode/intel/000306A9-02 | Bin 13312 -> 13312 bytes
 usr/src/data/ucode/intel/000306A9-10 | Bin 13312 -> 13312 bytes
 usr/src/data/ucode/intel/000306C3-02 | Bin 23552 -> 23552 bytes
 usr/src/data/ucode/intel/000306C3-10 | Bin 23552 -> 23552 bytes
 usr/src/data/ucode/intel/000306C3-20 | Bin 23552 -> 23552 bytes
 usr/src/data/ucode/intel/000306D4-40 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000306D4-80 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000306E4-01 | Bin 15360 -> 15360 bytes
 usr/src/data/ucode/intel/000306E4-04 | Bin 15360 -> 15360 bytes
 usr/src/data/ucode/intel/000306E4-08 | Bin 15360 -> 15360 bytes
 usr/src/data/ucode/intel/000306E4-20 | Bin 15360 -> 15360 bytes
 usr/src/data/ucode/intel/000306E4-40 | Bin 15360 -> 15360 bytes
 usr/src/data/ucode/intel/000306E4-80 | Bin 15360 -> 15360 bytes
 usr/src/data/ucode/intel/000306E7-01 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-04 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-08 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-20 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-40 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000306E7-80 | Bin 16384 -> 17408 bytes
 usr/src/data/ucode/intel/000306F2-01 | Bin 33792 -> 33792 bytes
 usr/src/data/ucode/intel/000306F2-02 | Bin 33792 -> 33792 bytes
 usr/src/data/ucode/intel/000306F2-04 | Bin 33792 -> 33792 bytes
 usr/src/data/ucode/intel/000306F2-08 | Bin 33792 -> 33792 bytes
 usr/src/data/ucode/intel/000306F2-20 | Bin 33792 -> 33792 bytes
 usr/src/data/ucode/intel/000306F2-40 | Bin 33792 -> 33792 bytes
 usr/src/data/ucode/intel/000306F4-80 | Bin 17408 -> 17408 bytes
 usr/src/data/ucode/intel/00040651-02 | Bin 21504 -> 22528 bytes
 usr/src/data/ucode/intel/00040651-10 | Bin 21504 -> 22528 bytes
 usr/src/data/ucode/intel/00040651-20 | Bin 21504 -> 22528 bytes
 usr/src/data/ucode/intel/00040651-40 | Bin 21504 -> 22528 bytes
 usr/src/data/ucode/intel/00040661-02 | Bin 25600 -> 25600 bytes
 usr/src/data/ucode/intel/00040661-10 | Bin 25600 -> 25600 bytes
 usr/src/data/ucode/intel/00040661-20 | Bin 25600 -> 25600 bytes
 usr/src/data/ucode/intel/00040671-02 | Bin 12288 -> 13312 bytes
 usr/src/data/ucode/intel/00040671-20 | Bin 12288 -> 13312 bytes
 usr/src/data/ucode/intel/000406E3-40 | Bin 99328 -> 99328 bytes
 usr/src/data/ucode/intel/000406E3-80 | Bin 99328 -> 99328 bytes
 usr/src/data/ucode/intel/000406F1-01 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/000406F1-02 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/000406F1-04 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/000406F1-08 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/000406F1-20 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/000406F1-40 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/000406F1-80 | Bin 26624 -> 28672 bytes
 usr/src/data/ucode/intel/00050654-01 | Bin 28672 -> 31744 bytes
 usr/src/data/ucode/intel/00050654-02 | Bin 28672 -> 31744 bytes
 usr/src/data/ucode/intel/00050654-04 | Bin 28672 -> 31744 bytes
 usr/src/data/ucode/intel/00050654-10 | Bin 28672 -> 31744 bytes
 usr/src/data/ucode/intel/00050654-20 | Bin 28672 -> 31744 bytes
 usr/src/data/ucode/intel/00050654-80 | Bin 28672 -> 31744 bytes
 usr/src/data/ucode/intel/00050662-10 | Bin 31744 -> 31744 bytes
 usr/src/data/ucode/intel/00050663-10 | Bin 22528 -> 22528 bytes
 usr/src/data/ucode/intel/00050664-10 | Bin 22528 -> 22528 bytes
 usr/src/data/ucode/intel/00050665-10 | Bin 18432 -> 18432 bytes
 usr/src/data/ucode/intel/000506C2-01 | Bin 0 -> 15360 bytes
 usr/src/data/ucode/intel/000506C9-01 | Bin 16384 -> 16384 bytes
 usr/src/data/ucode/intel/000506C9-02 | Bin 16384 -> 16384 bytes
 usr/src/data/ucode/intel/000506CA-01 | Bin 0 -> 14336 bytes
 usr/src/data/ucode/intel/000506CA-02 | Bin 0 -> 14336 bytes
 usr/src/data/ucode/intel/000506E3-02 | Bin 99328 -> 99328 bytes
 usr/src/data/ucode/intel/000506E3-04 | Bin 99328 -> 99328 bytes
 usr/src/data/ucode/intel/000506E3-10 | Bin 99328 -> 99328 bytes
 usr/src/data/ucode/intel/000506E3-20 | Bin 99328 -> 99328 bytes
 usr/src/data/ucode/intel/000506F1-01 | Bin 0 -> 10240 bytes
 usr/src/data/ucode/intel/000706A1-01 | Bin 72704 -> 73728 bytes
 usr/src/data/ucode/intel/000806E9-40 | Bin 98304 -> 98304 bytes
 usr/src/data/ucode/intel/000806E9-80 | Bin 98304 -> 98304 bytes
 usr/src/data/ucode/intel/000806EA-40 | Bin 97280 -> 98304 bytes
 usr/src/data/ucode/intel/000806EA-80 | Bin 97280 -> 98304 bytes
 usr/src/data/ucode/intel/000906E9-02 | Bin 98304 -> 98304 bytes
 usr/src/data/ucode/intel/000906E9-08 | Bin 98304 -> 98304 bytes
 usr/src/data/ucode/intel/000906E9-20 | Bin 98304 -> 98304 bytes
 usr/src/data/ucode/intel/000906EA-02 | Bin 96256 -> 97280 bytes
 usr/src/data/ucode/intel/000906EA-20 | Bin 96256 -> 97280 bytes
 usr/src/data/ucode/intel/000906EB-02 | Bin 98304 -> 98304 bytes
 104 files changed, 8 insertions(+), 1 deletion(-)
 create mode 100644 usr/src/data/ucode/intel/000206C2-01
 create mode 100644 usr/src/data/ucode/intel/000206C2-02
 create mode 100644 usr/src/data/ucode/intel/000206E6-04
 create mode 100644 usr/src/data/ucode/intel/000506C2-01
 create mode 100644 usr/src/data/ucode/intel/000506CA-01
 create mode 100644 usr/src/data/ucode/intel/000506CA-02
 create mode 100644 usr/src/data/ucode/intel/000506F1-01

diff --git a/manifest b/manifest
index ef9ad3d079..09f2eeeba8 100644
--- a/manifest
+++ b/manifest
@@ -2074,6 +2074,8 @@ f platform/i86pc/ucode/GenuineIntel/00020661-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00020661-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206A7-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206A7-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000206C2-01 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000206C2-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206D6-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206D6-04 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206D6-08 0444 root sys
@@ -2084,6 +2086,7 @@ f platform/i86pc/ucode/GenuineIntel/000206D7-04 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206D7-08 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206D7-20 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206D7-40 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000206E6-04 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206F2-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000206F2-04 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000306A9-02 0444 root sys
@@ -2151,12 +2154,16 @@ f platform/i86pc/ucode/GenuineIntel/00050662-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050663-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050664-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/00050665-10 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000506C2-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506C9-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506C9-02 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000506CA-01 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000506CA-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506E3-02 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506E3-04 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506E3-10 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000506E3-20 0444 root sys
+f platform/i86pc/ucode/GenuineIntel/000506F1-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000706A1-01 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000806E9-40 0444 root sys
 f platform/i86pc/ucode/GenuineIntel/000806E9-80 0444 root sys
diff --git a/usr/src/data/ucode/README.ucode b/usr/src/data/ucode/README.ucode
index 40552c546e..964c984da4 100644
--- a/usr/src/data/ucode/README.ucode
+++ b/usr/src/data/ucode/README.ucode
@@ -30,4 +30,4 @@ the release that the microcode was obtained from here.
 AMD: Updated in March 2012 as part of illumos#2546. Exact revision
 unknown.
 
-Intel: Linux 20180312 release
+Intel: Linux 20180807 release
diff --git a/usr/src/data/ucode/intel/000106A5-01 b/usr/src/data/ucode/intel/000106A5-01
index 22689fc795..154e08c826 100644
Binary files a/usr/src/data/ucode/intel/000106A5-01 and b/usr/src/data/ucode/intel/000106A5-01 differ
diff --git a/usr/src/data/ucode/intel/000106A5-02 b/usr/src/data/ucode/intel/000106A5-02
index 22689fc795..154e08c826 100644
Binary files a/usr/src/data/ucode/intel/000106A5-02 and b/usr/src/data/ucode/intel/000106A5-02 differ
diff --git a/usr/src/data/ucode/intel/000106E5-01 b/usr/src/data/ucode/intel/000106E5-01
index a8da6ed13e..b692f51dd2 100644
Binary files a/usr/src/data/ucode/intel/000106E5-01 and b/usr/src/data/ucode/intel/000106E5-01 differ
diff --git a/usr/src/data/ucode/intel/000106E5-02 b/usr/src/data/ucode/intel/000106E5-02
index a8da6ed13e..b692f51dd2 100644
Binary files a/usr/src/data/ucode/intel/000106E5-02 and b/usr/src/data/ucode/intel/000106E5-02 differ
diff --git a/usr/src/data/ucode/intel/000106E5-10 b/usr/src/data/ucode/intel/000106E5-10
index a8da6ed13e..b692f51dd2 100644
Binary files a/usr/src/data/ucode/intel/000106E5-10 and b/usr/src/data/ucode/intel/000106E5-10 differ
diff --git a/usr/src/data/ucode/intel/00020652-02 b/usr/src/data/ucode/intel/00020652-02
index 88bbeb2999..f63ee5581e 100644
Binary files a/usr/src/data/ucode/intel/00020652-02 and b/usr/src/data/ucode/intel/00020652-02 differ
diff --git a/usr/src/data/ucode/intel/00020652-10 b/usr/src/data/ucode/intel/00020652-10
index 88bbeb2999..f63ee5581e 100644
Binary files a/usr/src/data/ucode/intel/00020652-10 and b/usr/src/data/ucode/intel/00020652-10 differ
diff --git a/usr/src/data/ucode/intel/00020655-02 b/usr/src/data/ucode/intel/00020655-02
index 85a1a13d39..06bbbc501d 100644
Binary files a/usr/src/data/ucode/intel/00020655-02 and b/usr/src/data/ucode/intel/00020655-02 differ
diff --git a/usr/src/data/ucode/intel/00020655-10 b/usr/src/data/ucode/intel/00020655-10
index 85a1a13d39..06bbbc501d 100644
Binary files a/usr/src/data/ucode/intel/00020655-10 and b/usr/src/data/ucode/intel/00020655-10 differ
diff --git a/usr/src/data/ucode/intel/00020655-80 b/usr/src/data/ucode/intel/00020655-80
index 85a1a13d39..06bbbc501d 100644
Binary files a/usr/src/data/ucode/intel/00020655-80 and b/usr/src/data/ucode/intel/00020655-80 differ
diff --git a/usr/src/data/ucode/intel/000206A7-02 b/usr/src/data/ucode/intel/000206A7-02
index b321377e62..e20cdbf957 100644
Binary files a/usr/src/data/ucode/intel/000206A7-02 and b/usr/src/data/ucode/intel/000206A7-02 differ
diff --git a/usr/src/data/ucode/intel/000206A7-10 b/usr/src/data/ucode/intel/000206A7-10
index b321377e62..e20cdbf957 100644
Binary files a/usr/src/data/ucode/intel/000206A7-10 and b/usr/src/data/ucode/intel/000206A7-10 differ
diff --git a/usr/src/data/ucode/intel/000206C2-01 b/usr/src/data/ucode/intel/000206C2-01
new file mode 100644
index 0000000000..2a78b4a4d2
Binary files /dev/null and b/usr/src/data/ucode/intel/000206C2-01 differ
diff --git a/usr/src/data/ucode/intel/000206C2-02 b/usr/src/data/ucode/intel/000206C2-02
new file mode 100644
index 0000000000..2a78b4a4d2
Binary files /dev/null and b/usr/src/data/ucode/intel/000206C2-02 differ
diff --git a/usr/src/data/ucode/intel/000206D6-01 b/usr/src/data/ucode/intel/000206D6-01
index b22cc16ecb..d89a291172 100644
Binary files a/usr/src/data/ucode/intel/000206D6-01 and b/usr/src/data/ucode/intel/000206D6-01 differ
diff --git a/usr/src/data/ucode/intel/000206D6-04 b/usr/src/data/ucode/intel/000206D6-04
index b22cc16ecb..d89a291172 100644
Binary files a/usr/src/data/ucode/intel/000206D6-04 and b/usr/src/data/ucode/intel/000206D6-04 differ
diff --git a/usr/src/data/ucode/intel/000206D6-08 b/usr/src/data/ucode/intel/000206D6-08
index b22cc16ecb..d89a291172 100644
Binary files a/usr/src/data/ucode/intel/000206D6-08 and b/usr/src/data/ucode/intel/000206D6-08 differ
diff --git a/usr/src/data/ucode/intel/000206D6-20 b/usr/src/data/ucode/intel/000206D6-20
index b22cc16ecb..d89a291172 100644
Binary files a/usr/src/data/ucode/intel/000206D6-20 and b/usr/src/data/ucode/intel/000206D6-20 differ
diff --git a/usr/src/data/ucode/intel/000206D6-40 b/usr/src/data/ucode/intel/000206D6-40
index b22cc16ecb..d89a291172 100644
Binary files a/usr/src/data/ucode/intel/000206D6-40 and b/usr/src/data/ucode/intel/000206D6-40 differ
diff --git a/usr/src/data/ucode/intel/000206D7-01 b/usr/src/data/ucode/intel/000206D7-01
index a29a6f79ec..0da2b9e3f5 100644
Binary files a/usr/src/data/ucode/intel/000206D7-01 and b/usr/src/data/ucode/intel/000206D7-01 differ
diff --git a/usr/src/data/ucode/intel/000206D7-04 b/usr/src/data/ucode/intel/000206D7-04
index a29a6f79ec..0da2b9e3f5 100644
Binary files a/usr/src/data/ucode/intel/000206D7-04 and b/usr/src/data/ucode/intel/000206D7-04 differ
diff --git a/usr/src/data/ucode/intel/000206D7-08 b/usr/src/data/ucode/intel/000206D7-08
index a29a6f79ec..0da2b9e3f5 100644
Binary files a/usr/src/data/ucode/intel/000206D7-08 and b/usr/src/data/ucode/intel/000206D7-08 differ
diff --git a/usr/src/data/ucode/intel/000206D7-20 b/usr/src/data/ucode/intel/000206D7-20
index a29a6f79ec..0da2b9e3f5 100644
Binary files a/usr/src/data/ucode/intel/000206D7-20 and b/usr/src/data/ucode/intel/000206D7-20 differ
diff --git a/usr/src/data/ucode/intel/000206D7-40 b/usr/src/data/ucode/intel/000206D7-40
index a29a6f79ec..0da2b9e3f5 100644
Binary files a/usr/src/data/ucode/intel/000206D7-40 and b/usr/src/data/ucode/intel/000206D7-40 differ
diff --git a/usr/src/data/ucode/intel/000206E6-04 b/usr/src/data/ucode/intel/000206E6-04
new file mode 100644
index 0000000000..1ec0402183
Binary files /dev/null and b/usr/src/data/ucode/intel/000206E6-04 differ
diff --git a/usr/src/data/ucode/intel/000206F2-01 b/usr/src/data/ucode/intel/000206F2-01
index 8fe47d5333..8cebe97f86 100644
Binary files a/usr/src/data/ucode/intel/000206F2-01 and b/usr/src/data/ucode/intel/000206F2-01 differ
diff --git a/usr/src/data/ucode/intel/000206F2-04 b/usr/src/data/ucode/intel/000206F2-04
index 8fe47d5333..8cebe97f86 100644
Binary files a/usr/src/data/ucode/intel/000206F2-04 and b/usr/src/data/ucode/intel/000206F2-04 differ
diff --git a/usr/src/data/ucode/intel/000306A9-02 b/usr/src/data/ucode/intel/000306A9-02
index bb8a981f73..f965ba53d5 100644
Binary files a/usr/src/data/ucode/intel/000306A9-02 and b/usr/src/data/ucode/intel/000306A9-02 differ
diff --git a/usr/src/data/ucode/intel/000306A9-10 b/usr/src/data/ucode/intel/000306A9-10
index bb8a981f73..f965ba53d5 100644
Binary files a/usr/src/data/ucode/intel/000306A9-10 and b/usr/src/data/ucode/intel/000306A9-10 differ
diff --git a/usr/src/data/ucode/intel/000306C3-02 b/usr/src/data/ucode/intel/000306C3-02
index e76d3a017c..4dd6664e28 100644
Binary files a/usr/src/data/ucode/intel/000306C3-02 and b/usr/src/data/ucode/intel/000306C3-02 differ
diff --git a/usr/src/data/ucode/intel/000306C3-10 b/usr/src/data/ucode/intel/000306C3-10
index e76d3a017c..4dd6664e28 100644
Binary files a/usr/src/data/ucode/intel/000306C3-10 and b/usr/src/data/ucode/intel/000306C3-10 differ
diff --git a/usr/src/data/ucode/intel/000306C3-20 b/usr/src/data/ucode/intel/000306C3-20
index e76d3a017c..4dd6664e28 100644
Binary files a/usr/src/data/ucode/intel/000306C3-20 and b/usr/src/data/ucode/intel/000306C3-20 differ
diff --git a/usr/src/data/ucode/intel/000306D4-40 b/usr/src/data/ucode/intel/000306D4-40
index 97daace041..64d13670df 100644
Binary files a/usr/src/data/ucode/intel/000306D4-40 and b/usr/src/data/ucode/intel/000306D4-40 differ
diff --git a/usr/src/data/ucode/intel/000306D4-80 b/usr/src/data/ucode/intel/000306D4-80
index 97daace041..64d13670df 100644
Binary files a/usr/src/data/ucode/intel/000306D4-80 and b/usr/src/data/ucode/intel/000306D4-80 differ
diff --git a/usr/src/data/ucode/intel/000306E4-01 b/usr/src/data/ucode/intel/000306E4-01
index 59c4504ad1..c9792b7515 100644
Binary files a/usr/src/data/ucode/intel/000306E4-01 and b/usr/src/data/ucode/intel/000306E4-01 differ
diff --git a/usr/src/data/ucode/intel/000306E4-04 b/usr/src/data/ucode/intel/000306E4-04
index 59c4504ad1..c9792b7515 100644
Binary files a/usr/src/data/ucode/intel/000306E4-04 and b/usr/src/data/ucode/intel/000306E4-04 differ
diff --git a/usr/src/data/ucode/intel/000306E4-08 b/usr/src/data/ucode/intel/000306E4-08
index 59c4504ad1..c9792b7515 100644
Binary files a/usr/src/data/ucode/intel/000306E4-08 and b/usr/src/data/ucode/intel/000306E4-08 differ
diff --git a/usr/src/data/ucode/intel/000306E4-20 b/usr/src/data/ucode/intel/000306E4-20
index 59c4504ad1..c9792b7515 100644
Binary files a/usr/src/data/ucode/intel/000306E4-20 and b/usr/src/data/ucode/intel/000306E4-20 differ
diff --git a/usr/src/data/ucode/intel/000306E4-40 b/usr/src/data/ucode/intel/000306E4-40
index 59c4504ad1..c9792b7515 100644
Binary files a/usr/src/data/ucode/intel/000306E4-40 and b/usr/src/data/ucode/intel/000306E4-40 differ
diff --git a/usr/src/data/ucode/intel/000306E4-80 b/usr/src/data/ucode/intel/000306E4-80
index 59c4504ad1..c9792b7515 100644
Binary files a/usr/src/data/ucode/intel/000306E4-80 and b/usr/src/data/ucode/intel/000306E4-80 differ
diff --git a/usr/src/data/ucode/intel/000306E7-01 b/usr/src/data/ucode/intel/000306E7-01
index 43ffabf285..94415b1fa1 100644
Binary files a/usr/src/data/ucode/intel/000306E7-01 and b/usr/src/data/ucode/intel/000306E7-01 differ
diff --git a/usr/src/data/ucode/intel/000306E7-04 b/usr/src/data/ucode/intel/000306E7-04
index 43ffabf285..94415b1fa1 100644
Binary files a/usr/src/data/ucode/intel/000306E7-04 and b/usr/src/data/ucode/intel/000306E7-04 differ
diff --git a/usr/src/data/ucode/intel/000306E7-08 b/usr/src/data/ucode/intel/000306E7-08
index 43ffabf285..94415b1fa1 100644
Binary files a/usr/src/data/ucode/intel/000306E7-08 and b/usr/src/data/ucode/intel/000306E7-08 differ
diff --git a/usr/src/data/ucode/intel/000306E7-20 b/usr/src/data/ucode/intel/000306E7-20
index 43ffabf285..94415b1fa1 100644
Binary files a/usr/src/data/ucode/intel/000306E7-20 and b/usr/src/data/ucode/intel/000306E7-20 differ
diff --git a/usr/src/data/ucode/intel/000306E7-40 b/usr/src/data/ucode/intel/000306E7-40
index 43ffabf285..94415b1fa1 100644
Binary files a/usr/src/data/ucode/intel/000306E7-40 and b/usr/src/data/ucode/intel/000306E7-40 differ
diff --git a/usr/src/data/ucode/intel/000306E7-80 b/usr/src/data/ucode/intel/000306E7-80
index 43ffabf285..94415b1fa1 100644
Binary files a/usr/src/data/ucode/intel/000306E7-80 and b/usr/src/data/ucode/intel/000306E7-80 differ
diff --git a/usr/src/data/ucode/intel/000306F2-01 b/usr/src/data/ucode/intel/000306F2-01
index b2b0f8245f..d6f7d702fc 100644
Binary files a/usr/src/data/ucode/intel/000306F2-01 and b/usr/src/data/ucode/intel/000306F2-01 differ
diff --git a/usr/src/data/ucode/intel/000306F2-02 b/usr/src/data/ucode/intel/000306F2-02
index b2b0f8245f..d6f7d702fc 100644
Binary files a/usr/src/data/ucode/intel/000306F2-02 and b/usr/src/data/ucode/intel/000306F2-02 differ
diff --git a/usr/src/data/ucode/intel/000306F2-04 b/usr/src/data/ucode/intel/000306F2-04
index b2b0f8245f..d6f7d702fc 100644
Binary files a/usr/src/data/ucode/intel/000306F2-04 and b/usr/src/data/ucode/intel/000306F2-04 differ
diff --git a/usr/src/data/ucode/intel/000306F2-08 b/usr/src/data/ucode/intel/000306F2-08
index b2b0f8245f..d6f7d702fc 100644
Binary files a/usr/src/data/ucode/intel/000306F2-08 and b/usr/src/data/ucode/intel/000306F2-08 differ
diff --git a/usr/src/data/ucode/intel/000306F2-20 b/usr/src/data/ucode/intel/000306F2-20
index b2b0f8245f..d6f7d702fc 100644
Binary files a/usr/src/data/ucode/intel/000306F2-20 and b/usr/src/data/ucode/intel/000306F2-20 differ
diff --git a/usr/src/data/ucode/intel/000306F2-40 b/usr/src/data/ucode/intel/000306F2-40
index b2b0f8245f..d6f7d702fc 100644
Binary files a/usr/src/data/ucode/intel/000306F2-40 and b/usr/src/data/ucode/intel/000306F2-40 differ
diff --git a/usr/src/data/ucode/intel/000306F4-80 b/usr/src/data/ucode/intel/000306F4-80
index a7df2e775b..f5967028bd 100644
Binary files a/usr/src/data/ucode/intel/000306F4-80 and b/usr/src/data/ucode/intel/000306F4-80 differ
diff --git a/usr/src/data/ucode/intel/00040651-02 b/usr/src/data/ucode/intel/00040651-02
index 3ee3568662..62a2dc9d13 100644
Binary files a/usr/src/data/ucode/intel/00040651-02 and b/usr/src/data/ucode/intel/00040651-02 differ
diff --git a/usr/src/data/ucode/intel/00040651-10 b/usr/src/data/ucode/intel/00040651-10
index 3ee3568662..62a2dc9d13 100644
Binary files a/usr/src/data/ucode/intel/00040651-10 and b/usr/src/data/ucode/intel/00040651-10 differ
diff --git a/usr/src/data/ucode/intel/00040651-20 b/usr/src/data/ucode/intel/00040651-20
index 3ee3568662..62a2dc9d13 100644
Binary files a/usr/src/data/ucode/intel/00040651-20 and b/usr/src/data/ucode/intel/00040651-20 differ
diff --git a/usr/src/data/ucode/intel/00040651-40 b/usr/src/data/ucode/intel/00040651-40
index 3ee3568662..62a2dc9d13 100644
Binary files a/usr/src/data/ucode/intel/00040651-40 and b/usr/src/data/ucode/intel/00040651-40 differ
diff --git a/usr/src/data/ucode/intel/00040661-02 b/usr/src/data/ucode/intel/00040661-02
index b6f2ffd13f..ee019bb34b 100644
Binary files a/usr/src/data/ucode/intel/00040661-02 and b/usr/src/data/ucode/intel/00040661-02 differ
diff --git a/usr/src/data/ucode/intel/00040661-10 b/usr/src/data/ucode/intel/00040661-10
index b6f2ffd13f..ee019bb34b 100644
Binary files a/usr/src/data/ucode/intel/00040661-10 and b/usr/src/data/ucode/intel/00040661-10 differ
diff --git a/usr/src/data/ucode/intel/00040661-20 b/usr/src/data/ucode/intel/00040661-20
index b6f2ffd13f..ee019bb34b 100644
Binary files a/usr/src/data/ucode/intel/00040661-20 and b/usr/src/data/ucode/intel/00040661-20 differ
diff --git a/usr/src/data/ucode/intel/00040671-02 b/usr/src/data/ucode/intel/00040671-02
index 0b1effa643..ed8a37ce73 100644
Binary files a/usr/src/data/ucode/intel/00040671-02 and b/usr/src/data/ucode/intel/00040671-02 differ
diff --git a/usr/src/data/ucode/intel/00040671-20 b/usr/src/data/ucode/intel/00040671-20
index 0b1effa643..ed8a37ce73 100644
Binary files a/usr/src/data/ucode/intel/00040671-20 and b/usr/src/data/ucode/intel/00040671-20 differ
diff --git a/usr/src/data/ucode/intel/000406E3-40 b/usr/src/data/ucode/intel/000406E3-40
index f112757568..96a70f69b7 100644
Binary files a/usr/src/data/ucode/intel/000406E3-40 and b/usr/src/data/ucode/intel/000406E3-40 differ
diff --git a/usr/src/data/ucode/intel/000406E3-80 b/usr/src/data/ucode/intel/000406E3-80
index f112757568..96a70f69b7 100644
Binary files a/usr/src/data/ucode/intel/000406E3-80 and b/usr/src/data/ucode/intel/000406E3-80 differ
diff --git a/usr/src/data/ucode/intel/000406F1-01 b/usr/src/data/ucode/intel/000406F1-01
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-01 and b/usr/src/data/ucode/intel/000406F1-01 differ
diff --git a/usr/src/data/ucode/intel/000406F1-02 b/usr/src/data/ucode/intel/000406F1-02
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-02 and b/usr/src/data/ucode/intel/000406F1-02 differ
diff --git a/usr/src/data/ucode/intel/000406F1-04 b/usr/src/data/ucode/intel/000406F1-04
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-04 and b/usr/src/data/ucode/intel/000406F1-04 differ
diff --git a/usr/src/data/ucode/intel/000406F1-08 b/usr/src/data/ucode/intel/000406F1-08
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-08 and b/usr/src/data/ucode/intel/000406F1-08 differ
diff --git a/usr/src/data/ucode/intel/000406F1-20 b/usr/src/data/ucode/intel/000406F1-20
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-20 and b/usr/src/data/ucode/intel/000406F1-20 differ
diff --git a/usr/src/data/ucode/intel/000406F1-40 b/usr/src/data/ucode/intel/000406F1-40
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-40 and b/usr/src/data/ucode/intel/000406F1-40 differ
diff --git a/usr/src/data/ucode/intel/000406F1-80 b/usr/src/data/ucode/intel/000406F1-80
index 8c9f97f007..ff10f9a341 100644
Binary files a/usr/src/data/ucode/intel/000406F1-80 and b/usr/src/data/ucode/intel/000406F1-80 differ
diff --git a/usr/src/data/ucode/intel/00050654-01 b/usr/src/data/ucode/intel/00050654-01
index a6415d4a8f..dda53a919d 100644
Binary files a/usr/src/data/ucode/intel/00050654-01 and b/usr/src/data/ucode/intel/00050654-01 differ
diff --git a/usr/src/data/ucode/intel/00050654-02 b/usr/src/data/ucode/intel/00050654-02
index a6415d4a8f..dda53a919d 100644
Binary files a/usr/src/data/ucode/intel/00050654-02 and b/usr/src/data/ucode/intel/00050654-02 differ
diff --git a/usr/src/data/ucode/intel/00050654-04 b/usr/src/data/ucode/intel/00050654-04
index a6415d4a8f..dda53a919d 100644
Binary files a/usr/src/data/ucode/intel/00050654-04 and b/usr/src/data/ucode/intel/00050654-04 differ
diff --git a/usr/src/data/ucode/intel/00050654-10 b/usr/src/data/ucode/intel/00050654-10
index a6415d4a8f..dda53a919d 100644
Binary files a/usr/src/data/ucode/intel/00050654-10 and b/usr/src/data/ucode/intel/00050654-10 differ
diff --git a/usr/src/data/ucode/intel/00050654-20 b/usr/src/data/ucode/intel/00050654-20
index a6415d4a8f..dda53a919d 100644
Binary files a/usr/src/data/ucode/intel/00050654-20 and b/usr/src/data/ucode/intel/00050654-20 differ
diff --git a/usr/src/data/ucode/intel/00050654-80 b/usr/src/data/ucode/intel/00050654-80
index a6415d4a8f..dda53a919d 100644
Binary files a/usr/src/data/ucode/intel/00050654-80 and b/usr/src/data/ucode/intel/00050654-80 differ
diff --git a/usr/src/data/ucode/intel/00050662-10 b/usr/src/data/ucode/intel/00050662-10
index 83ef3a6ec7..44c2a7f731 100644
Binary files a/usr/src/data/ucode/intel/00050662-10 and b/usr/src/data/ucode/intel/00050662-10 differ
diff --git a/usr/src/data/ucode/intel/00050663-10 b/usr/src/data/ucode/intel/00050663-10
index 546734c86d..bdf73e86c9 100644
Binary files a/usr/src/data/ucode/intel/00050663-10 and b/usr/src/data/ucode/intel/00050663-10 differ
diff --git a/usr/src/data/ucode/intel/00050664-10 b/usr/src/data/ucode/intel/00050664-10
index dcb28b5dd6..d2b33205f7 100644
Binary files a/usr/src/data/ucode/intel/00050664-10 and b/usr/src/data/ucode/intel/00050664-10 differ
diff --git a/usr/src/data/ucode/intel/00050665-10 b/usr/src/data/ucode/intel/00050665-10
index 757943cab3..2b38ee5d70 100644
Binary files a/usr/src/data/ucode/intel/00050665-10 and b/usr/src/data/ucode/intel/00050665-10 differ
diff --git a/usr/src/data/ucode/intel/000506C2-01 b/usr/src/data/ucode/intel/000506C2-01
new file mode 100644
index 0000000000..387e1e23f7
Binary files /dev/null and b/usr/src/data/ucode/intel/000506C2-01 differ
diff --git a/usr/src/data/ucode/intel/000506C9-01 b/usr/src/data/ucode/intel/000506C9-01
index eb879064e1..a9739ad0b5 100644
Binary files a/usr/src/data/ucode/intel/000506C9-01 and b/usr/src/data/ucode/intel/000506C9-01 differ
diff --git a/usr/src/data/ucode/intel/000506C9-02 b/usr/src/data/ucode/intel/000506C9-02
index eb879064e1..a9739ad0b5 100644
Binary files a/usr/src/data/ucode/intel/000506C9-02 and b/usr/src/data/ucode/intel/000506C9-02 differ
diff --git a/usr/src/data/ucode/intel/000506CA-01 b/usr/src/data/ucode/intel/000506CA-01
new file mode 100644
index 0000000000..792380734f
Binary files /dev/null and b/usr/src/data/ucode/intel/000506CA-01 differ
diff --git a/usr/src/data/ucode/intel/000506CA-02 b/usr/src/data/ucode/intel/000506CA-02
new file mode 100644
index 0000000000..792380734f
Binary files /dev/null and b/usr/src/data/ucode/intel/000506CA-02 differ
diff --git a/usr/src/data/ucode/intel/000506E3-02 b/usr/src/data/ucode/intel/000506E3-02
index 301019a41a..6e2cd0f68d 100644
Binary files a/usr/src/data/ucode/intel/000506E3-02 and b/usr/src/data/ucode/intel/000506E3-02 differ
diff --git a/usr/src/data/ucode/intel/000506E3-04 b/usr/src/data/ucode/intel/000506E3-04
index 301019a41a..6e2cd0f68d 100644
Binary files a/usr/src/data/ucode/intel/000506E3-04 and b/usr/src/data/ucode/intel/000506E3-04 differ
diff --git a/usr/src/data/ucode/intel/000506E3-10 b/usr/src/data/ucode/intel/000506E3-10
index 301019a41a..6e2cd0f68d 100644
Binary files a/usr/src/data/ucode/intel/000506E3-10 and b/usr/src/data/ucode/intel/000506E3-10 differ
diff --git a/usr/src/data/ucode/intel/000506E3-20 b/usr/src/data/ucode/intel/000506E3-20
index 301019a41a..6e2cd0f68d 100644
Binary files a/usr/src/data/ucode/intel/000506E3-20 and b/usr/src/data/ucode/intel/000506E3-20 differ
diff --git a/usr/src/data/ucode/intel/000506F1-01 b/usr/src/data/ucode/intel/000506F1-01
new file mode 100644
index 0000000000..83b441b9bc
Binary files /dev/null and b/usr/src/data/ucode/intel/000506F1-01 differ
diff --git a/usr/src/data/ucode/intel/000706A1-01 b/usr/src/data/ucode/intel/000706A1-01
index 67d5c8ba1d..6fe711f7ce 100644
Binary files a/usr/src/data/ucode/intel/000706A1-01 and b/usr/src/data/ucode/intel/000706A1-01 differ
diff --git a/usr/src/data/ucode/intel/000806E9-40 b/usr/src/data/ucode/intel/000806E9-40
index 7e4b899891..c03eb315a3 100644
Binary files a/usr/src/data/ucode/intel/000806E9-40 and b/usr/src/data/ucode/intel/000806E9-40 differ
diff --git a/usr/src/data/ucode/intel/000806E9-80 b/usr/src/data/ucode/intel/000806E9-80
index 7e4b899891..c03eb315a3 100644
Binary files a/usr/src/data/ucode/intel/000806E9-80 and b/usr/src/data/ucode/intel/000806E9-80 differ
diff --git a/usr/src/data/ucode/intel/000806EA-40 b/usr/src/data/ucode/intel/000806EA-40
index 96f38c08f3..641d07ef43 100644
Binary files a/usr/src/data/ucode/intel/000806EA-40 and b/usr/src/data/ucode/intel/000806EA-40 differ
diff --git a/usr/src/data/ucode/intel/000806EA-80 b/usr/src/data/ucode/intel/000806EA-80
index 96f38c08f3..641d07ef43 100644
Binary files a/usr/src/data/ucode/intel/000806EA-80 and b/usr/src/data/ucode/intel/000806EA-80 differ
diff --git a/usr/src/data/ucode/intel/000906E9-02 b/usr/src/data/ucode/intel/000906E9-02
index 73be2bb208..a1cc285d60 100644
Binary files a/usr/src/data/ucode/intel/000906E9-02 and b/usr/src/data/ucode/intel/000906E9-02 differ
diff --git a/usr/src/data/ucode/intel/000906E9-08 b/usr/src/data/ucode/intel/000906E9-08
index 73be2bb208..a1cc285d60 100644
Binary files a/usr/src/data/ucode/intel/000906E9-08 and b/usr/src/data/ucode/intel/000906E9-08 differ
diff --git a/usr/src/data/ucode/intel/000906E9-20 b/usr/src/data/ucode/intel/000906E9-20
index 73be2bb208..a1cc285d60 100644
Binary files a/usr/src/data/ucode/intel/000906E9-20 and b/usr/src/data/ucode/intel/000906E9-20 differ
diff --git a/usr/src/data/ucode/intel/000906EA-02 b/usr/src/data/ucode/intel/000906EA-02
index 80a042652e..f4475ceea7 100644
Binary files a/usr/src/data/ucode/intel/000906EA-02 and b/usr/src/data/ucode/intel/000906EA-02 differ
diff --git a/usr/src/data/ucode/intel/000906EA-20 b/usr/src/data/ucode/intel/000906EA-20
index 80a042652e..f4475ceea7 100644
Binary files a/usr/src/data/ucode/intel/000906EA-20 and b/usr/src/data/ucode/intel/000906EA-20 differ
diff --git a/usr/src/data/ucode/intel/000906EB-02 b/usr/src/data/ucode/intel/000906EB-02
index c4773bc76b..e0c1b96628 100644
Binary files a/usr/src/data/ucode/intel/000906EB-02 and b/usr/src/data/ucode/intel/000906EB-02 differ
-- 
2.21.0

